omod/src/main/resources/config.xml

```diff
@@ -56,6 +56,29 @@
 		<url-pattern>/ms/fhir2R3Servlet/*</url-pattern>
 	</filter-mapping>
 
+	<!--filter for adding CORS support-->
+	<!-- custom filter available in org.openmrs.module.oauth2.web.util-->
+	<filter>
+		<filter-name>CorsFilter</filter-name>
+		<filter-class>org.openmrs.module.smartonfhir.web.filter.CORSFilter</filter-class>
+	</filter>
+	<filter-mapping>
+		<filter-name>CorsFilter</filter-name>
+		<url-pattern>/*</url-pattern>
```

Comment:
Can we make this a little narrower? Ideally something like `/ws/fhir2/*` or even `/ws/fhir2/*/.well-known`.

Reply:
@ibacher I don't think that the CORS filter is working as we would expect it to work because even after enabling this I could see a CORS policy error in console.

Could you test it a bit?

Reply:
@ibacher I just tested the CORS filter again and its working perfectly alright.